packages/devtools_app/lib/src/screens/memory/panes/diff/controller/diff_pane_controller.dart

```diff
@@ -31,11 +31,13 @@ import 'snapshot_item.dart';
 
 // ignore: avoid_classes_with_only_static_members, enum-like classes are ok
 @visibleForTesting
-class Json {
-  static const snapshots = 'snapshots';
-  static const diffWith = 'diffWith';
+enum Json {
+  snapshots('snapshots'),
+  diffWith('diffWith');
 
-  static final all = [snapshots, diffWith];
+  const Json(this.key);
+
+  final String key;
```

Comment:
unnecessary. Json.snapshots.name and Json.diffWith.name are equivalent. This comment applies here and elsewhere to other enums.